### DISCUSSION TITLE
第一个小游戏章节，存在误导/表述不清

### DISCUSSION THREAD

**Discussion by Godow** [galacean/discussion#89] [Editor] [answered]
引擎文档，第一个小游戏。"可以使用编辑器的同学直接使用编辑器进行搭建。"表示可以用galacean编辑器来做这个小游戏。下面代码引用tween时候，形式为"from "@tweenjs/tween.js";"，这很容易误导使用的npm，但是galacean编辑器又不支持npm。

**Comment by eyworldwide** [comment]
@Godow 感谢你的反馈，教程正在重构中  cc: @cptbtptpbcptdtptp

**Comment by cptbtptpbcptdtptp** [comment]
已经优化文档中依赖外部库部分的描述。

**Comment by eyworldwide** [comment]
@Godow 我们已经重构了一版 2D 游戏教学文档：https://galacean.antgroup.com/#/docs/latest/cn/editor-first-2d-project

**Comment by singlecoder** [comment]
最新版本的编辑器已经支持引入第三方包

**Comment by cptbtptpbcptdtptp** [comment, marked as answer]
- 第一个问题，现在的 flappy bird 教程已经重构过一遍，其中原来依赖 tween 库的逻辑已经用编辑器自带的动画流程解决了，可以参考[像素小鸟](https://galacean.antgroup.com/engine/docs/latest/cn/getting-started-flappy-bird)
- 第二个问题，如 @singlecoder  所说的，最新版本的编辑器已经支持引入第三方包，可以参考[编辑脚本](https://galacean.antgroup.com/engine/docs/latest/cn/script-edit#%E5%BC%95%E5%85%A5%E7%AC%AC%E4%B8%89%E6%96%B9%E5%8C%85)
